.github/workflows/deploy-staging.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy backend to staging
+
+on:
+  push:
+    branches: ["development"]
+  pull_request:
```

Comment:
Doesn't this mean that it runs whenever a pull request is created? Shouldn't it only trigger if the pull request is merged?